.bcr/config.yml

```diff
@@ -0,0 +1,3 @@
+fixedReleaser:
+  login: Bencodes
+  email: ben@ben.cm
```

Comment:
@ted-xie or @ahumesky one of you probably needs to be listed here since I assume you have the permissions to cut release.

Reply:
made an edit